change/@microsoft-teams-js-385a0467-ba06-419f-842e-89301a76c1e8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added `{namespace}` capability that will {explain capability}. The capability is still awaiting support in one or most host applications. To track availability of this capability across different hosts see https://aka.ms/capmatrix",
```

Comment:
This comment gets turned into patch notes for external developers. That means you should fill in the name of the new capability as well as a brief description of what it is for.